packages/orchestration/src/examples/stakeBld.contract.js

```diff
@@ -34,48 +40,86 @@ export const start = async (zcf, privateArgs, baggage) => {
     baggage,
     privateArgs.marshaller,
   );
-  const makeLocalchainAccountKit = prepareLocalchainAccountKit(
+
+  // Mocked until #8879
+  // Would expect this to be instantiated elsewhere, and passed in as a reference
+  const agoricChainInfo = prepareMockChainInfo(zone);
+
+  const timerBrand = await E(privateArgs.timerService).getTimerBrand();
+  const timestampHelper = makeTimestampHelper(
+    privateArgs.timerService,
+    timerBrand,
+  );
+
+  const makeLocalChainAccountKit = prepareLocalChainAccountKit(
     baggage,
     makeRecorderKit,
     zcf,
+    timestampHelper,
+    agoricChainInfo,
   );
 
-  const publicFacet = zone.exo('StakeBld', undefined, {
-    makeStakeBldInvitation() {
-      return zcf.makeInvitation(
-        async seat => {
-          const { give } = seat.getProposal();
-          trace('makeStakeBldInvitation', give);
-          // XXX type appears local but it's remote
-          const account = await E(privateArgs.localchain).makeAccount();
-          const lcaSeatKit = zcf.makeEmptySeatKit();
-          atomicTransfer(zcf, seat, lcaSeatKit.zcfSeat, give);
-          seat.exit();
-          trace('makeStakeBldInvitation tryExit lca userSeat');
-          await E(lcaSeatKit.userSeat).tryExit();
-          trace('awaiting payouts');
-          const payouts = await E(lcaSeatKit.userSeat).getPayouts();
-          const { holder, invitationMakers } = makeLocalchainAccountKit(
-            account,
-            privateArgs.storageNode,
-          );
-          trace('awaiting deposit');
-          await E(account).deposit(await payouts.In);
+  async function makeLocalAccount() {
```

Comment:
as noted earlier:

```suggestion
  async function makeLocalAccountKit() {
```

cf docs on [Method Naming Structure](https://docs.agoric.com/guides/ertp/#method-naming-structure)